test/unittests/mocks/Dbus_interface_mock.h

```diff
@@ -1,17 +1,15 @@
-#ifndef SDBUS_C_DBUS_INTERFACE_MOCK_H
-#define SDBUS_C_DBUS_INTERFACE_MOCK_H
+#ifndef SDBUS_CXX_DBUS_INTERFACE_MOCK_H
+#define SDBUS_CXX_DBUS_INTERFACE_MOCK_H
 
-#include <gmock/gmock.h>
+#include "ISdBus.h"
 
-#include "IDbus.h"
+#include <gmock/gmock.h>
 
-class Dbus_interface_mock : public IDbus
+class Dbus_interface_mock : public ISdBus
 {
 public:
     Dbus_interface_mock() {};
     Dbus_interface_mock(const Dbus_interface_mock& other) {};
-    virtual ~Dbus_interface_mock() {};
```

Comment:
Good that you've removed an unnecessary explicit d-tor. I propose the same for the two c-tors here. They provide no real logic in their implementation and thus will be perfectly well-generated by the compiler, and the code will be shorter and less cluttered. (See the very nice Rule of Five / the Rule of All or Nothing by Scott Meyers / Arne Mertz about explicit c-tors, copy c-tors, move c-tors and d-tors).